spec/models/actor_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe Actor, type: :model do
+  describe "relationships" do
+    it {should have_many(:movies).through(:movie_actors)}
+  end
+
+  describe "methods" do 
```

Comment:
I'm guessing you set this up preemptively for something that we ended up not needing. Typically we should leave out code we aren't actually using, because in case we never come back to this, it just becomes garbage in our code that we forget about and is a huge cleanup project later. (Not calling your code garbage - just the idea of unused garbage being trash that needs to be cleaned up)